lib/bundle.js

```diff
@@ -87,7 +89,7 @@ exports.bundle = function(moduleExpression, fileName, opts) {
 
     if (opts.inject) {
       // Add the bundle to config if the inject flag was given.
-      var bundleName = extractBundleName(fileName);
+      var bundleName = extractBundleName(fileName, systemBuilder);
```

Comment:
Oh right, sorry of course. That sounds fine then. I generally like inlining when something isn't used elsewhere as I find it more readable, but happy either way on this.